storm-server/src/main/java/org/apache/storm/DaemonConfig.java

```diff
@@ -307,6 +307,12 @@
     @isString
     public static final String UI_CENTRAL_LOGGING_URL = "ui.central.logging.url";
 
+    /**
+     * Storm UI drop-down pagination value.
```

Comment:
Could you also add in some example values that can be set in the javadocs.  Just so people know what to do.

Reply:
Done!